api/v1/Makefile.protoc

```diff
@@ -21,13 +21,20 @@ HUBBLE_PROTO_PATH := .
 HUBBLE_PROTOC_PLUGINS := --plugin=$(GOPATH)/bin/protoc-gen-doc
 HUBBLE_PROTOC_PLUGINS += --plugin=$(GOPATH)/bin/protoc-gen-go-grpc
 HUBBLE_PROTOC_PLUGINS += --plugin=$(GOPATH)/bin/protoc-gen-go-json
+HUBBLE_PROTOC_PLUGINS += --plugin=$(GOPATH)/bin/protoc-gen-gotag
 HUBBLE_PROTOC_PLUGINS += --plugin=$(GOPATH)/bin/protoc-gen-go
 HUBBLE_PROTOC_PLUGINS += --plugin=$(GOPATH)/bin/protoc-gen-deepcopy
 
 .PHONY: all
 all:
 	@echo NOTE: The warning about package github.com/golang/protobuf/protoc-gen-go/generator can be ignored
 	$(QUIET)set -e; \
+	# The protoc-gen-gotag operates on already generated go file and requires it
+	# to be available within target go package. Symlink is used for files to be
+	# resolvable.
+	rm -fr github.com && \
```

Comment:
This looks incredibly dangerous. Anyone running this accidentally outside the build container is potentially going to loose data.

Can we instead fix up the mount path in the `Makefile`?

Reply:
fixed